### PR TITLE
Created JSON file containing English text from all dialogs

### PR DESCRIPTION
**Pull request by lloyddewit**
Related to (but does not fix) issue #3721 . 

@africanmathsinitiative/developers There is nothing you can review or test in this PR. This PR is just a new English source file for the CrowdIn translations.

@dannyparsons This new English JSON file should include the text from `frmMain` and all the dialogs. This is the vast majority of the English text in R-Instat. However it is still missing some non-standard elements:

- the `frmMain` menus (this text is in `r_instat_menus.json` (when the initial translations are done, I suggest we combine the 'menu' and 'non-menu' JSON files into a single JSON file)
- right mouse button menus in the `frmMain` windows (e.g. script window)
- tool bar pull down menus
- `ucrCheck` and `ucrInput` controls (these controls use dynamic text, for later implementation)
- text in message boxes
- any other dynamic text
- text containing carriage returns or line feeds (causes problems for the csv files that I use to populate the database)

I will probably find more exceptions as I test. 
The code I use to generate the English language JSON file(s) will be included in a future PR.

Please could the CrowdIn translations of this JSON file into French be done in parallel to my work on the translation mechanism?
Please may I add extra English text to this file in a future PR (for the missing cases above)?
Thanks if this is possible.

### CODE REVIEW

**Comment by lloyddewit** [comment]
@dannyparsons I added new English JSON files. As discussed, `r_instat_menus.json` contains all the text associated with `frmMain`, and `r_instat_not_menus.json` contains all the other text.

I tried to include all the static and dynamic text in the controls. Some text is not included:

- text in message boxes
- text containing carriage returns or line feeds (causes problems for the csv files that I use to populate the database)
- composite text (e.g. built up of different elements such as text including path names, row numbers, counts etc.)

File: instat/translations/en/r_instat_not_menus.json [hunk 1]
```diff
@@ -1 +1,2396 @@
...
+"Describe Display Options": "Describe Display Options", 
+"Describe Survival": "Describe Survival", 
+"Describe Two Variables": "Describe Two Variables", 
+"Details Options": "Details Options", 
+"Detect": "Detect", 
+"Difference Column:": "Difference Column:", 
+"Differences": "Differences", 
+"Dimensions to Plot:": "Dimensions to Plot:", 
+"Direction (N/S E/W):": "Direction (N/S E/W):", 
+"Dispaly Daily Options": "Dispaly Daily Options",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:480]: I can see some typo here

File: instat/translations/en/r_instat_not_menus.json [hunk 2]
```diff
@@ -1 +1,2396 @@
...
+"Fit an Extreme Distribution": "Fit an Extreme Distribution", 
+"Fitted Model:": "Fitted Model:", 
+"Fitting Method": "Fitting Method", 
+"Fitting Options": "Fitting Options", 
+"Fixed": "Fixed", 
+"Fixed Day": "Fixed Day", 
+"Fixed Number of Components": "Fixed Number of Components", 
+"Fixed width": "Fixed width", 
+"Flat Frequency Table": "Flat Frequency Table", 
+"Fliter Preview:": "Fliter Preview:",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:673]: Also here

File: instat/translations/en/r_instat_not_menus.json [hunk 3]
```diff
@@ -1 +1,2396 @@
...
+"Labels/Levels": "Labels/Levels", 
+"Lang": "Lang", 
+"Language": "Language", 
+"Languages": "Languages", 
+"Last": "Last", 
+"Last Word:": "Last Word:", 
+"Latitude": "Latitude", 
+"Latitude Offset:": "Latitude Offset:", 
+"Latitude:": "Latitude:", 
+"Latitute:": "Latitute:",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:887]: also here

File: instat/translations/en/r_instat_not_menus.json [hunk 4]
```diff
@@ -1 +1,2396 @@
...
+"Max Longitude:": "Max Longitude:", 
+"Max Missing Days": "Max Missing Days", 
+"Max:": "Max:", 
+"Maximum": "Maximum", 
+"Maximum Distance from 0": "Maximum Distance from 0", 
+"Maximum Dry Days:": "Maximum Dry Days:", 
+"Maximum Factor Levels Shown:": "Maximum Factor Levels Shown:", 
+"Maximum Latitude": "Maximum Latitude", 
+"Maximum Likelihood (MLE)": "Maximum Likelihood (MLE)", 
+"Maximum Logitude": "Maximum Logitude",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1004]: also here

File: instat/translations/en/r_instat_not_menus.json [hunk 5]
```diff
@@ -1 +1,2396 @@
...
+"PICSA Crops": "PICSA Crops", 
+"PICSA Options": "PICSA Options", 
+"PICSA Rainfall Graph": "PICSA Rainfall Graph", 
+"PICSA Rainfall Graphs": "PICSA Rainfall Graphs", 
+"PICSA Temperature": "PICSA Temperature", 
+"PP": "PP", 
+"PP-Plot": "PP-Plot", 
+"PPP Adjusted Contract Value:": "PPP Adjusted Contract Value:", 
+"Package Name:": "Package Name:", 
+"Packages Documenation": "Packages Documenation",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1244]: also here

File: instat/translations/en/r_instat_not_menus.json [hunk 6]
```diff
@@ -1 +1,2396 @@
...
+"Plus:": "Plus:", 
+"Point": "Point", 
+"Point Jitter:": "Point Jitter:", 
+"Point Options": "Point Options", 
+"Point Transparency:": "Point Transparency:", 
+"Points": "Points", 
+"Polar Annulus": "Polar Annulus", 
+"Polar Cluster": "Polar Cluster", 
+"Polar Coordinate:": "Polar Coordinate:", 
+"Polar Cordinates": "Polar Cordinates",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1314]: also here

File: instat/translations/en/r_instat_not_menus.json [hunk 7]
```diff
@@ -1 +1,2396 @@
...
+"Pollutant:": "Pollutant:", 
+"Polygon Options": "Polygon Options", 
+"Polynomial:": "Polynomial:", 
+"Polynomials": "Polynomials", 
+"Population Pyramids": "Population Pyramids", 
+"Port:": "Port:", 
+"Position": "Position", 
+"Position - Estimates:": "Position - Estimates:", 
+"Position:": "Position:", 
+"Postion:": "Postion:",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1326]: also here

File: instat/translations/en/r_instat_not_menus.json [hunk 8]
```diff
@@ -1 +1,2396 @@
...
+"SS 2009:": "SS 2009:", 
+"SS 2011:": "SS 2011:", 
+"SS 2013": "SS 2013", 
+"SS 2015:": "SS 2015:", 
+"SST Data Frame Name:": "SST Data Frame Name:", 
+"Sample Size:": "Sample Size:", 
+"Sampling Fraction:": "Sampling Fraction:", 
+"Satelite": "Satelite", 
+"Satelite Anomalies": "Satelite Anomalies", 
+"Satelite Element:": "Satelite Element:",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1499]: Should all these be `satellite`?

File: instat/translations/en/r_instat_not_menus.json [hunk 9]
```diff
@@ -1 +1,2396 @@
...
+"Smoothing Curves": "Smoothing Curves", 
+"Solar": "Solar", 
+"Sort": "Sort", 
+"Sort by Row Names": "Sort by Row Names", 
+"Source:": "Source:", 
+"Space between Strips and Axes": "Space between Strips and Axes", 
+"Span:": "Span:", 
+"Spanish": "Spanish", 
+"Spearman": "Spearman", 
+"Specifiy": "Specifiy",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_not_menus.json:1652]: also here

File: instat/translations/en/r_instat_menus.json [hunk 10]
```diff
@@ -1,408 +1,429 @@
...
+"  Log Window...": "  Log Window...",
+"  Script Window...": "  Script Window...",
+"&Copy": "&Copy",
+"&Open": "&Open",
+"&Paste": "&Paste",
+"&Save": "&Save",
+"ANOVA General...": "ANOVA General...",
+"About R": "About R",
+"About R-Instat": "About R-Instat",
+"Acknowledgements": "Acknowledgements",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_menus.json:12]: also here? I remember you changed this in the menu recently

File: instat/translations/en/r_instat_menus.json [hunk 11]
```diff
@@ -1,408 +1,429 @@
...
+"Correlations (Red Flags or others)...": "Correlations (Red Flags or others)...",
+"Correlations...": "Correlations...",
+"Corruption Risk Index": "Corruption Risk Index",
+"Count Records...": "Count Records...",
+"Count in Factor...": "Count in Factor...",
+"Create Climate Object...": "Create Climate Object...",
+"Crop": "Crop",
+"Crop Coefficients...": "Crop Coefficients...",
+"Crops...": "Crops...",
+"Cummulative Exceedance...": "Cummulative Exceedance...",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_menus.json:87]: `cumulative`?

File: instat/translations/en/r_instat_menus.json [hunk 12]
```diff
@@ -1,408 +1,429 @@
...
+"Missing Data...": "Missing Data...",
+"Missing Values...": "Missing Values...",
+"Model": "Model",
+"Model...": "Model...",
+"Monthly Summaries...": "Monthly Summaries...",
+"More...": "More...",
+"Mosaic Plot...": "Mosaic Plot...",
+"Multiple Response...": "Multiple Response...",
+"Multivariate": "Multivariate",
+"Multple Lines...": "Multple Lines...",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_menus.json:236]: also here

File: instat/translations/en/r_instat_menus.json [hunk 13]
```diff
@@ -1,408 +1,429 @@
...
+"NCMP": "NCMP",
+"New Data Frame...": "New Data Frame...",
+"New Worksheet...": "New Worksheet...",
+"No data loaded": "No data loaded",
+"Non Parameteric One Way ANOVA...": "Non Parameteric One Way ANOVA...",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_menus.json:241]: `Parametric` ?

File: instat/translations/en/r_instat_menus.json [hunk 14]
```diff
@@ -1,408 +1,429 @@
...
+"Themes...": "Themes...",
+"Three Summaries...": "Three Summaries...",
+"Three Variables": "Three Variables",
+"Three Variables...": "Three Variables...",
+"Tidy Daily Data...": "Tidy Daily Data...",
+"Tidy and Examine": "Tidy and Examine",
+"Tidy...": "Tidy...",
+"Time Series": "Time Series",
+"Time Series Plot...": "Time Series Plot...",
+"Timeseries...": "Timeseries...",
```
Comment (shadrackkibet) [inline instat/translations/en/r_instat_menus.json:393]: Should this be two words?

**Review by shadrackkibet — CHANGES_REQUESTED** [review]
@lloyddewit since there is nothing to test, I only checked the spelling mistakes in the file.

**Comment by lloyddewit** [comment]
@shadrackkibet Thank you for spotting the typos.
The JSON file in this PR reflects the text in the `.vb` and `.resx` files. Therefore we need to fix the typos in these `.vb` and `.resx` files, and then generate a new translations database and translation JSON files.
I think we can best do this after version 0.7.1 is released.
I'd prefer to do this in a separate PR. I made issue PR #6438 for this.

**Comment by dannyparsons** [comment]
Thanks Stephen, I will merge this and start getting the translations done in CrowdIn.

@shadrackkibet thanks for spotting the typos. Can we try to find where these are in forms/dialogs and correct them there? This will then be reflected in the json files as Stephen mentioned.

**Review by dannyparsons — APPROVED** [review]

**Pull request merged**